ONSdigital/ras-frontstage
# Added cookies control functionality and pages

# Motivation and Context
- Added cookies control functionality to the service. 
- Note - This is a temporary implementation and will be refactored with the service as a whole.

# What has changed
- Cookies banner added to `base` template to display across all pages.
- Frontstage cookies added to cookies js under 'essential' category (cannot be controlled by user).
- Cookies page - JS driven page for control of cookie categories.
- All cookies (except essential) are off by default.
- New pages added /cookies & /privacy-and-data-protection.

# How to test?
- View the banner.
- Clicking accept all cookies button will sett all non essential cookies to "on".
- Secondary banner should display.
- Go to /cookies and change values of radios to test the cookies are set correctly.
- Check cookies (in browser cookies inspector) to check essential cookies are being set ok.
